cranelift/codegen/src/machinst/lower.rs

```diff
@@ -607,7 +608,20 @@ impl<'func, I: VCodeInst> Lower<'func, I> {
                     ));
                 }
             }
-            if let Some(insn) = self.vcode.abi().gen_retval_area_setup(self.sigs()) {
+            if let Some(insn) = self
+                .vcode
+                .vcode
+                .abi
+                .gen_retval_area_setup(&self.vcode.vcode.sigs)
+            {
+                self.emit(insn);
+            }
+
+            // The `args` instruction below must come first. Finish the
+            // current "IR inst" and continue the scan backward.
+            self.finish_ir_inst(Default::default());
```

Comment:
I guess as long as I'm asking for more things... something like this? It wasn't obvious to me what the "default" value here meant.

```suggestion
            // The `args` instruction below must come first. Finish the
            // current "IR inst" with a default source location and continue the scan backward.
            self.finish_ir_inst(Default::default());
```

Reply:
Done!